src/signals/incident-management/containers/IncidentDetail/components/StatusForm/styled.tsx

```diff
@@ -2,85 +2,114 @@
 // Copyright (C) 2020 - 2021 Gemeente Amsterdam
 import styled from 'styled-components'
 import {
+  Alert,
   Heading,
-  themeSpacing,
+  Label,
   Row,
-  Column,
   themeColor,
-  Label,
+  themeSpacing,
 } from '@amsterdam/asc-ui'
 
 import Button from 'components/Button'
 
+export const AddNoteWrapper = styled.div`
+  label {
+    display: none;
+  }
+  section div textarea {
+    padding: ${themeSpacing(3)};
+    margin-top: 0;
+    border-top: 1px solid transparent;
+    :hover {
+      border-top-color: transparent;
+    }
+  }
+`
+
 export const Form = styled.form`
   position: relative;
+  padding: ${themeSpacing(0, 5, 6, 5)};
+  margin-bottom: ${themeSpacing(6)};
+  background-color: ${themeColor('tint', 'level2')};
   width: 100%;
   grid-template-areas:
     'header'
     'options'
     'texts'
     'form';
```

Comment:
Does this form still need grid layout? For what I've seen, it's merely a vertical layout now.

Reply:
A vertical grid? with vertical spacing?

Reply:
removed